torchsynth/synth.py

```diff
@@ -112,7 +112,7 @@ def test_step(self, batch, batch_idx):
         # We just return 0, which lightning accumulates as the test error
         return T(0.0, device=self.device)
 
-    def randomize(self, seed: Optional[int]):
+    def randomize(self, seed: Optional[int] = None):
```

Comment:
Noticing the `if seed` statement below -- it should be updated to `if seed is not None:`

Reply:
oops will fix

Reply:
fixed